e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/OrderTest.groovy

```diff
@@ -25,18 +25,6 @@ class OrderTest extends Specification {
         parsedJsonBody.patientId == expectedPatientId
     }
 
-    def "payload file check"() {
-        when:
-        def response = orderClient.submit(labOrderJsonFileString, true)
-        def sentPayload = SentPayloadReader.read()
-        def parsedSentPayload = JsonParsing.parse(sentPayload)
-        def parsedLabOrderJsonFile = JsonParsing.parse(labOrderJsonFileString)
-
-        then:
-        response.getCode() == 200
-        parsedSentPayload == parsedLabOrderJsonFile
```

Comment:
File was deleted because we only check for the status of 200, which is being done in another test.